src/types/events/base-events.ts

```diff
@@ -425,6 +426,42 @@ export interface FileUnsharedEvent {
   event_ts: string;
 }
 
+export interface FunctionParams {
+  type?: string;
+  name?: string;
+  description?: string;
+  title?: string;
+  is_required?: boolean;
+}
+
+export interface FunctionInputs {
+  [key: string]: unknown;
+}
+
+export type FunctionOutputValues = FunctionInputs;
+
+export interface FunctionExecutedEvent {
+  type: 'function_executed';
+  function: {
+    id: string;
+    callback_id: string;
+    title: string;
+    description: string;
```

Comment:
```suggestion
    description?: string;
```

Reply:
Thanks! Also included in [pre-stable-fixes branch](https://github.com/slackapi/bolt-js/pull/2128/files#diff-7e67aa7d0c6331b94638639b911c10002ef75e4b5be11e75449ee92f3d2c7623R454).